src/main/java/org/candlepin/util/X509V3ExtensionUtil.java

```diff
@@ -409,7 +406,7 @@ private Branding getBranding(Pool pool, String productId) {
             // Check if we should override the enabled flag due to setting on promoted content
             if (enableEnvironmentFiltering && consumer.getEnvironmentId() != null) {
```

Comment:
The `getEnvironmentId()` method is to be removed in the next task (SCA cert generation), so we should change this check to instead be `!consumer.getEnvironmentIds().isEmpty()` instead